server/jetty/src/main/java/io/deephaven/server/jetty/ControlledCacheResource.java

```diff
@@ -0,0 +1,122 @@
+package io.deephaven.server.jetty;
+
+import org.eclipse.jetty.util.resource.Resource;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.nio.channels.ReadableByteChannel;
+
+/**
+ * Simple wrapper around the Jetty Resource type, to grant us control over caching features. The current implementation
+ * only removes the last-modified value, but a future version could provide a "real" weak/strong etag.
+ */
+public class ControlledCacheResource extends Resource {
+    private final Resource wrapped;
+
+    public ControlledCacheResource(Resource wrapped) {
```

Comment:
Seems like there's a lot of methods we're not overriding, such as `.toURL`, `.getURL`, `.setAssociate`, etc. Are we sure these are the only functions that need to be overridden? How did you decide what to override?

Reply:
I only overrode the abstract methods, and only modified the one that was specific to last-modified. The rest of the methods are implemented by the superclass Resource to rely on the abstract methods. 

 * setAssociate is kind of a special case - it isnt clear to me how it is used, and has explicit docs that it isn't actually used...
 * isSame already delegates to equals (allowing for a subclass to override isSame separately), but we already override equals
 * getResource delegates to addPath, which we override
 * isAlias calls getAlias, and getAlias has a sane default of null for resources that dont alias anything. It looks like this is meant for soft links, absolute path vs canonical path, but we're only dealing with paths inside of jars, so it shouldnt ever matter.
 * getListHTML generates HTML and has a default impl that we don't want to own
 * copyTo arguably does need us to implement it differently to work properly, but at the same time, we don't really support this - internally it is used for unpacking web archives to read them off of the disk
 * getAllResources delegates to list() and addPath(), both of which we implement, no other implementation overrides this default impl
 * 

That said, toURL is static, so doesnt need to be overridden, and getURL _is_ overridden in this class (since it is abstract).

Reply:
Nit-pick - `getURI()` is overridden, not `getURL()`... but looks like `getURL` is deprecated anyway so I think that's okay.